chemistrylab/extract_algorithms/extractions/methyl_red.py

```diff
@@ -113,7 +115,7 @@ def reset(self, extraction_vessel):
 
         ethyl = material.EthylAcetate
```

Comment:
Would it make more sense to make this an object rather than doing it multiple times later?

Reply:
I think you could. The reason I didn't was to make the change the same across the different gyms I updated